docs/plugins/trello-github-integ_plugin.md

```diff
@@ -37,13 +24,51 @@ tools:
     owner: YOUR_GITHUB_USERNAME
     # the repo where you'd like to setup GitHub Actions; please change the value below.
     repo: YOUR_REPO_NAME
-    # integration tool name
-    api:
-      name: trello
-      # name of the Trello kanban board
-      kanbanBoardName: kanban-name
+    # reference parameters come from dependency, their usage will be explained later
+    boardId: ${{ default.trello.outputs.boardId }}
+    todoListId: ${{ default.trello.outputs.todoListId }}
+    doingListId: ${{ default.trello.outputs.doingListId }}
+    doneListId: ${{ default.trello.outputs.doneListId }}
     # main branch of the repo (to which branch the plugin will submit the workflows)
     branch: main
 ```
 
-Currently, all the parameters in the example above are mandatory.
+## 3. Use Together with the `trello` Plugin
+
+This plugin can be used together with the `trello` plugin (see document [here](./trello_plugin.md).)
+
+See the example below:
+
+```yaml
+---
+tools:
+  - name: trello_init_demo
```

Comment:
The name is `trello_init_demo` here, but I saw the `default.trello` at line 70/71.

So the name is `default` or `trello_init_demo`?